src/main.cpp

```diff
@@ -644,6 +644,7 @@ bool IsStandardTx(const CTransaction& tx, string& reason)
     return true;
 }
 
+bool CheckFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime);
```

Comment:
This declaration will be necessary later anyway when the function is moved in https://github.com/jtimon/bitcoin/commit/6769c02ca34303648de74f42c7dfd0cb1aaa41da#diff-cbe22f30d7e480617350ef6ceca97d0cR65 
So it's a temporary thing, and, yes, we have function declarations in main.cpp, for example, these two: https://github.com/bitcoin/bitcoin/blob/master/src/main.cpp#L81
The reason   to make it in this way is to make the diff smaller.